app/mailers/patient_mailer.rb

```diff
@@ -38,9 +38,16 @@ def enrollment_sms_text_based(patient)
 
     lang = patient.select_language(:sms)
     custom_msg = patient&.jurisdiction&.custom_messages_for_hierarchy&.dig('assessments', 'twilio', 'sms', 'prompt', 'intro', lang&.to_s)
-    contents = custom_msg.present? ?
-      "#{patient&.initials_age('-')},\n#{custom_msg}" :
-      I18n.t('assessments.twilio.sms.prompt.intro', locale: lang, name: patient&.initials_age('-'))
+    contents = if custom_msg.present?
+                 "#{patient&.initials_age('-')},\n#{custom_msg}" +
+                   (ADMIN_OPTIONS['privacy_policy_url'].present? ?
+                     I18n.t('assessments.shared.privacy_info', locale: lang, privacy_policy_url: ADMIN_OPTIONS['privacy_policy_url']) : '')
+               else
+                 I18n.t('assessments.twilio.sms.prompt.intro', locale: lang, name: patient&.initials_age('-')) +
+                   (ADMIN_OPTIONS['privacy_policy_url'].present? ?
+                     ' ' + I18n.t('assessments.shared.privacy_info', locale: lang, privacy_policy_url: ADMIN_OPTIONS['privacy_policy_url']) : '')
+               end
+    puts contents
```

Comment:
Debugging statement left in

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1247/commits/a597631babae41a24088894361513966e30256b4